### PR TITLE
Tabs render without array options

### PR DESCRIPTION
**Pull request by hitaloramon**
### What:

- [ ] Bug Fix
- [x] Improvement
- [ ] New Feature
- [ ] Test Coverage

### Description:

Added the possibility for tabs to render automatically without the need to inform the options array

### CODE REVIEW

File: resources/js/modules/components/tabs/tabs.js [hunk 1]
```diff
@@ -1,9 +1,15 @@
 export default (selected) => ({
   tab: selected ?? null,
+  tabHeadings: [],
```
Comment (devajmeireles) [inline resources/js/modules/components/tabs/tabs.js:3]: Could we rename this to `headings`?

File: resources/views/components/tabs/index.blade.php [hunk 2]
```diff
@@ -18,18 +18,17 @@
...
-                @class($customize['item.wrapper'])
-                x-on:click="select(@js($tab))"
-                x-bind:aria-selected="selected(@js($tab))"
-                x-bind:class="selected(@js($tab)) ? '{{ $customize['item.selected']}}' : '{{ $customize['item.unselected']}}'"
+        <template x-for="tabItem in tabHeadings">
+            <li @class($customize['item.wrapper'])
+                x-on:click="select(tabItem)"
+                x-bind:aria-selected="selected(tabItem)"
+                x-bind:class="selected(tabItem) ? '{{ $customize['item.selected']}}' : '{{ $customize['item.unselected']}}'"
+                x-text="tabItem"
```
Comment (devajmeireles) [inline resources/views/components/tabs/index.blade.php:26]: After renaming to `headings` this code should be enough.

```suggestion
        <template x-for="item in headings">
            <li @class($customize['item.wrapper'])
                x-on:click="select(item)"
                x-bind:aria-selected="selected(item)"
                x-bind:class="selected(item) ? '{{ $customize['item.selected']}}' : '{{ $customize['item.unselected']}}'"
                x-text="item"
```

File: resources/views/components/tabs/items.blade.php [hunk 3]
```diff
@@ -1,5 +1,5 @@
 @php($customize = tallstackui_personalization('tabs.items', $customization()))
 
-<div x-show="selected(@js($tab))" role="tabpanel" {{ $attributes->merge(['class' => $customize['item']]) }}>
+<div x-data="{name: '{{$tab}}'}" x-show="selected(@js($tab))" role="tabpanel" {{ $attributes->merge(['class' => $customize['item']]) }}>
```
Comment (devajmeireles) [inline resources/views/components/tabs/items.blade.php:3]: Couldn't we do something like this?

```
x-data={ name : @js($tab) ... }
```

**Review by devajmeireles — CHANGES_REQUESTED** [review]

File: resources/views/components/tabs/items.blade.php (hunk 3 above)
Comment (hitaloramon) [reply resources/views/components/tabs/items.blade.php:3]: > Couldn't we do something like this?
> 
> ```
> x-data={ name : @js($tab) ... }
> ```

yes, also works

**Review by devajmeireles — APPROVED** [review]

**Pull request merged**